domaintools/api.py

```diff
@@ -65,7 +65,17 @@ def _rate_limit(self):
         """Pulls in and enforces the latest rate limits for the specified user"""
         self.limits_set = True
         for product in self.account_information():
-            self.limits[product['id']] = {'interval': timedelta(seconds=60 / float(product['per_minute_limit']))}
+            limit_minutes = product['per_minute_limit']
+            limit_hours = product['per_hour_limit']
```

Comment:
@CAMIRO this is failing against the production API endpoint since we don't have a per_hour_limit set there. I suggest we check for that and treat it as optional, setting `limit_hours` to `None` if its not in the API response:

```
>>> a._rate_limit()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/dnunes/Documents/GitHub/python_api/domaintools/api.py", line 69, in _rate_limit
    limit_hours = product['per_hour_limit']
KeyError: 'per_hour_limit'
```